### PR TITLE
feat(brew): Add brew leaves command

### PR DESCRIPTION
**Pull request by Owpac**
## Description

New script command for brew: `brew leaves`
List installed formulae that are not dependencies of another installed formula.

## Type of change

- [x] New script command

## Screenshot

Like brew list command

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

(Sorry I edited commands/README file)

### CODE REVIEW

File: commands/README.md [hunk 1]
```diff
@@ -1034,4 +1035,4 @@ This is a shared place and we're always looking for new Script Commands or other
 If you have anything cool to show, please send us a pull request. If we screwed something up,
 please report a bug. Join our
 [Slack community](https://www.raycast.com/community)
-to brainstorm ideas with like-minded folks.
\ No newline at end of file
+to brainstorm ideas with like-minded folks.
```
Comment (unnamedd) [inline commands/README.md:1038]: Please, revert the changes done in this file, it is auto-generated and all your changes here will be lost.

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @Owpac, 
just one small comment regarding the README.md, please address it and let us know when you finish.

Thanks for your first contribution!

File: commands/README.md (hunk 1 above)
Comment (Owpac) [reply commands/README.md:1037]: It should be good, sorry for this mistake !

Comment (unnamedd) [reply commands/README.md:1037]: Would you mind to revert this line as well? 
Thanks a lot and sorry for all these requests!

Comment (Owpac) [reply commands/README.md]: Done :)

**Review by unnamedd — APPROVED** [review]
Looks good to me! 
Thanks for your changes @Owpac!

**Comment by RSO** [comment]
Thank you for your contribution!

**Comment by Owpac** [comment]
@unnamedd I noticed a mistake in @raycast.description: "Show Brew List" --> "Show Brew Leaves"
Should I submit a new PR or I let you commit this update?

Sorry for the inconvenience!

**Comment by RSO** [comment]
I'll make the change, no worries!

**Comment by RSO** [comment]
2c74d4ee83bb00fe415cc73e0a65f9d922a4682c I took the description from `brew -h leaves`.

**Comment by Owpac** [comment]
Thanks! :)

**Comment by unnamedd** [comment]
> I'll make the change, no worries!

Nice! Thanks for the fix Remon!

**Pull request merged**